packages/env/src/index.ts

```diff
@@ -49,6 +57,8 @@ export const env = createEnv({
     S3_BUCKET: z.string().optional(),
     AWS_ACCESS_KEY: z.string().optional(),
     AWS_ACCESS_SECRET: z.string().optional(),
+    FILE_PUBLIC_PATH: z.string(),
+    FILES_STORAGE_PATH: z.string(),
```

Comment:
why two envvars?

Reply:
One is `/uploads` or whatever in case we need public URLs (for dev or people using filesystem). The other is disk path